src/HighscoreUpdater.java

```diff
@@ -0,0 +1,78 @@
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import static java.lang.Integer.parseInt;
+
+public class HighscoreUpdater {
+
+    public List<HighscoreEntry> readHighscores() {
+        List<HighscoreEntry> highscores = new ArrayList<>();
+        try (Scanner scan = new Scanner(new FileReader("Highscores.csv"))) {
+            while (scan.hasNext()) {
+                String scoreData = scan.next();
+                String[] scoreValues = scoreData.split(",");
+                String playerName = scoreValues[0];
+                int playerScore = parseInt(scoreValues[1]);
+                char difficultyLevel = scoreValues[2].charAt(0);
+                HighscoreEntry highscoreEntry = new HighscoreEntry(playerName,playerScore,difficultyLevel);
+                highscores.add(highscoreEntry);
+            }
+        } catch (IOException ioe) {
+            System.err.println("Something went wrong");
+        }
+        return highscores;
+    }
+
+    public List<HighscoreEntry> compareScore(Player player, char difficultyLevel, List<HighscoreEntry> highscores) {
+        int score = player.getPlayerScore();
+        boolean isHighscore = false;
+        for (int i = highscores.size() - 1 ; i >= 0; i--) {
+            if (!player.getPlayerName().equals("the computer")) {
```

Comment:
Als de naam van de speler "the computer" is, moet ie toch nooit aan de lijst toegevoegd worden? Dan zou ik die check als allereerste in je methode uitvoeren en niet in elke iteratie over de lijst (en als de naam "the computer" is, gewoon de oorspronkelijke lijst highscores teruggeven)